docs/proposals/EdgeSite.md

```diff
@@ -0,0 +1,78 @@
+---
+title: EdgeSite Design
+status: Pending
+authors: 
+  - "@cindyxing"
+approvers:
+  - "@qizha"
+  - "@kevin-wangzefeng"
+  - "@m1093782566"
+---
+# EdgeSite: Lightweight Cluster at edge
+
+## Abstract
+In Edge computing, there are scenarios where customers would like to have an whole cluster installed at edge location. As a result, 
+admins/users can leverage the local control plane to implement management functionalities and take advantages of all edge computing's benefits. 
+
+This design doc is to enable customers deploy and run lightweight clusters at edge. 
+
+## Motivation
+For these use cases, when considering large set of edge locations, the resource usage of management plane adding together will be high. 
+K3s is an opensource project enabling lightweight kubernetes cluster including the light control plane and worker components. 
+KubeEdge is the other opensource project enabling lightweight k8s client on the edge nodes. The KubeEdge agent runtime memory footprint is 
+10MB and it supports kubernetes types. Meanwhile KubEdge is targeted for Edge/IOT computing with many functionality enablements. 
+
+By integrating KubeEdge and K3s, this proposal is to allow customers to run an efficient kubernetes cluster for Edge/IOT computing. 
```

Comment:
I don't think edgesite should decouple with K3s, K3s can be one of the control plane backend and K8s can be another one as well. So we can create a more open framework for integrating K8s/ligthweight K8s and K3s?

PS. In our test, K3S Server still consume 300-400 MB memory.

Reply:
@m1093782566 I think you mean the edgesite solution should not be coupled with K3s.

Reply:
Edgesite is not coupled with K3S. I can update the wording a bit. 
The new edgesite client just watches a API server. The server can be k3s or a normal k8s.